src/main.cpp

```diff
@@ -4281,18 +4284,12 @@ void UnloadBlockIndex()
     mempool.clear();
     mapOrphanTransactions.clear();
     mapOrphanTransactionsByPrev.clear();
-    nSyncStarted = 0;
```

Comment:
Am I correct in assuming that everything removed here will eventually be making its way into PeerLogicValidation?